packages/core-web/package.json

```diff
@@ -22,8 +22,8 @@
   },
   "scripts": {
     "build": "npm run build:client && npm run build:server",
-    "build:client": "cross-env ENTRY_POINT=client webpack --progress --hide-modules --config webpack.build.js",
-    "build:server": "cross-env ENTRY_POINT=server webpack --progress --hide-modules --config webpack.build.js"
+    "build:client": "cross-env ENTRY_POINT=client webpack --progress --config webpack.build.js",
+    "build:server": "cross-env ENTRY_POINT=server webpack --progress --config webpack.build.js"
```

Comment:
Just curious, why are we removing the `hide-modules` option here (and what will the effect be)?

Reply:
I should have replaced the flag to `--stats.modules=false`.

That flag is fully deprecated and no logn supported. It fails our netlify deployment. This flag hides the output of modules details in the console. We should be able to replaced it with `--stats.modules=false`.

Reply:
Thanks @EltonGohJH ! Remember to update the proposed commit message (in the issue description) as well :)

Reply:
@EltonGohJH the commit message should be in the format of as described in the PR template comment:
> See this link for more info on how to write a good commit message:
  https://se-education.org/guides/conventions/git.html

@MarkBind/active-devs please take note for all your PRs.